app/scripts/app.js

```
@@ -337,6 +336,65 @@
   $rootScope.forceFullscreen = false;
 });
 
+function isIp(host) {
+  const ipComponents = host.split('.');
+  if (ipComponents.length !== 4) {
+    return false;
+  }
+  return ipComponents.every(num => {
+    const parsed = parseInt(num, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)